### PR TITLE
Issues #29 & #30: Fixed Typo in CONTRIBUTING.md and Linked to README.md

### PR DESCRIPTION
**Pull request by faokryn**
Fixed the formatting typo in CONTRIBUTING.md to fix the Markdown link. This resolves issue #29.

Rewrote the Contributing section of README.md, linking and directing potential contributors to CONTRIBUTING.md. This resolves issue #30.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ Software Citation Tools is a project proposed and overseen by Mozilla Science La
 Even though the team heading the project is doing this as a senior project, anyone is free to contribute! The RIT Software Engineering department has given approval for anyone to help out in any way without affecting the team's grade. Please feel free to contribute in any way you want!
 
 ##Contributing
-Right now we're trying to build a set of requirements and features by getting responses to some [questions](https://github.com/mozillascience/software-citation-tools/issues?q=is%3Aissue+is%3Aopen+label%3ADiscussion) we made.
+We encourage you to contriubte to this project in any way you can! An important goal of this project is to foster a community around these tools. If you'd like to be a part of that community, read our [contributing guide](https://github.com/mozillascience/software-citation-tools/blob/master/CONTRIBUTING.md) for more information.
```
Comment (Ourobor) [inline README.md]: typo with "contriubte"

Maybe make note that people only need to read the contributing guide if they want to add code. Adding to the discussion should be free form and easy for anyone

Comment (faokryn) [reply README.md]: The contributing guide has more information on contributing to the discussion. I think it's a good idea to send them there regardless.

Comment (Ourobor) [reply README.md]: Thumbs up looks good

**Comment by sam1360** [comment]
👍

**Comment by Ourobor** [comment]
Thumbs up You're doing great

**Pull request merged**